### PR TITLE
Code minified

### PR DESCRIPTION
**Pull request by lmfresneda**
- code minified (index.js)
- added contributer (package.json)
- replace HH:MM:ss with HH:mm:ss (README.md)

### CODE REVIEW

**Comment by lmfresneda** [comment]
I don't know why Travis fails :(

```
$ npm install 
/home/travis/.nvm/versions/node/v0.12.18/lib/node_modules/npm/lib/config/cmd-list.js:113
module.exports.aliases = Object.assign({}, shorthands, affordances)
                                ^
TypeError: undefined is not a function
    at Object.<anonymous> (/home/travis/.nvm/versions/node/v0.12.18/lib/node_modules/npm/lib/config/cmd-list.js:113:33)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at /home/travis/.nvm/versions/node/v0.12.18/lib/node_modules/npm/lib/npm.js:37:17
    at Object.<anonymous> (/home/travis/.nvm/versions/node/v0.12.18/lib/node_modules/npm/lib/npm.js:471:3)
    at Module._compile (module.js:460:26)
```

File: index.js [hunk 1]
```diff
@@ -2,50 +2,20 @@
...
-module.exports = log;
-module.exports.info = info;
-module.exports.dir = dir;
-module.exports.warn = warn;
-module.exports.error = error;
+module.exports = function(){ return _log(arguments, 'log'); };
+module.exports.info = function(){ return _log(arguments, 'info'); };
+module.exports.dir = function(){ return _log(arguments, 'dir'); };
+module.exports.warn = function(){ return _log(arguments, 'warn'); };
+module.exports.error = function(){ return _log(arguments, 'error'); };
```
Comment (jonschlinkert) [inline index.js:21]: I am really not a fan of this pattern for a couple reasons:

1. This abstraction makes the code harder to understand
2. It's not a good practice to [leaking arguments](https://stackoverflow.com/questions/30234908/javascript-v8-optimisation-and-leaking-arguments)

File: index.js [hunk 2]
```diff
@@ -2,50 +2,20 @@
 /*
   Initial code from https://github.com/gulpjs/gulp-util/blob/v3.0.6/lib/log.js
  */
-var chalk = require('chalk');
+var grey = require('chalk').grey;
```
Comment (jonschlinkert) [inline index.js:5]: if we only need gray, maybe we should consider using [ansi-gray](https://www.npmjs.com/package/ansi-gray)?

**Review by jonschlinkert — COMMENTED** [review]
I want to get feedback from @phated or @tkellen first, but my own preference is to avoid unnecessary abstractions. Either way, I love seeing pull requests, and I appreciate that you spent your valuable time on this library. Thank you.

**Comment by jonschlinkert** [comment]
> I don't know why Travis fails :(

The error message says that `Object.assign` is undefined. The `.assign` method was introduced in ES 2015, but the node engine in package.json of this project is `>= 0.10.0`

**Comment by lmfresneda** [comment]
Hi! Thanks for the quick reply!

It is true that it is not necessary, but I find it interesting because the (little) code does not need exhaustive maintenance. 

About the error, why it happens? I don't use 'Object.assign'... or I'm missing something...

**Comment by phated** [comment]
Thanks for the PR but we're going to pass on this.

**Pull request closed without merging**